labelbox/schema/ontology.py

```diff
@@ -151,21 +156,31 @@ def from_dict(cls, dictionary: Dict[str, Any]) -> Dict[str, Any]:
                    required=dictionary.get("required", False),
                    options=[Option.from_dict(o) for o in dictionary["options"]],
                    schema_id=dictionary.get("schemaNodeId", None),
-                   feature_schema_id=dictionary.get("featureSchemaId", None))
+                   feature_schema_id=dictionary.get("featureSchemaId", None),
+                   scope=cls.Scope(dictionary.get("scope", cls.Scope.GLOBAL)))
 
     def asdict(self) -> Dict[str, Any]:
         if self.class_type in self._REQUIRES_OPTIONS \
                 and len(self.options) < 1:
             raise InconsistentOntologyException(
                 f"Classification '{self.instructions}' requires options.")
         return {
-            "type": self.class_type.value,
-            "instructions": self.instructions,
-            "name": self.name,
-            "required": self.required,
+            "type":
+                self.class_type.value,
+            "instructions":
+                self.instructions,
+            "name":
+                self.name,
+            "required":
+                self.required,
             "options": [o.asdict() for o in self.options],
-            "schemaNodeId": self.schema_id,
-            "featureSchemaId": self.feature_schema_id
+            "schemaNodeId":
+                self.schema_id,
+            "featureSchemaId":
+                self.feature_schema_id,
+            "scope":
+                self.scope.value
+                if self.scope is not None else self.Scope.GLOBAL.value
```

Comment:
So we always want to include global scope by default? This won't cause any problems?

Reply:
Yeah, when talking with Amanda about this, for new ontologies it should be okay to default to global, since global is inclusive of image/videos while index is specific to videos

I am unsure how to update the code to check for this though:
```We also shouldn't allow users to update the scope if classifications annotations exist for that classification.```